cumulusci/utils/git.py

```diff
@@ -71,6 +72,9 @@ def parse_repo_url(url: str) -> Tuple[str, str, str]:
     Tuple: (str, str, str)
         Returns (owner, name, host)
     """
+    if not url:
+        raise error("Url is none or must have `remote` set as origin")
```

Comment:
±: Instead of using `from os import error`, consider using Python's built-in `ValueError` for validating the URL. It's more semantically correct for this use case.